Forms/CrcSettingsForm.cs

```diff
@@ -20,6 +20,7 @@ public partial class CrcSettingsForm : Form
         public CrcSettingsForm()
         {
             InitializeComponent();
+            this.changedConfigs = new Dictionary<string, dynamic>();

```

Comment:
no no no... use the getAndReset function...
this feels like a workaround

Reply:
```diff
diff --git a/Forms/CrcSettingsForm.cs b/Forms/CrcSettingsForm.cs
index 7b41113..4835f89 100755
--- a/Forms/CrcSettingsForm.cs
+++ b/Forms/CrcSettingsForm.cs
@@ -20,6 +20,8 @@ namespace CRCTray
         public CrcSettingsForm()
         {
             InitializeComponent();
+^M
+            getConfigurationAndResetChanged();^M
         }

         private void getConfigurationAndResetChanged()
@@ -38,8 +40,6 @@ namespace CRCTray
             Text = @"CodeReady Containers - Settings";
             this.FormClosing += CrcSettingsForm_FormClosing;
             this.tabs.SelectedIndexChanged += tabs_SelectedIndexChanged;
-
-            getConfigurationAndResetChanged();
         }

         private void tabs_SelectedIndexChanged(object sender, EventArgs e)
```

Reply:
Fixed, using the function now instead of directly accessing the variable